### DISCUSSION TITLE
导出pdf时侧边栏没有自动生成的序号

### DISCUSSION THREAD

**Discussion by sf88ux** [Keldos-Li/typora-latex-theme#168] [Q&A] [answered]
导出pdf后，正文包含自动生成的序号，但侧边栏没有自动生成的序号，能否解决一下，谢谢

**Comment by Keldos-Li** [comment, marked as answer]
试过了，解决不了，抱歉。

如果你一定要解决，可以**在标题处手动编号**，然后在导出时让Typora移除由样式自动添加的编号。方法是：
在设置中导出选项中，对PDF的导出选项，插入额外内容出添加如下代码：
```html
<script>
const styleSheet = document.styleSheets[0];
styleSheet.insertRule('h1::before, h2::before, h3::before, h4::before, h5::before, h6::before { content: none !important; }', styleSheet.cssRules.length);
</script>
```
<img width="721" alt="image" src="https://github.com/user-attachments/assets/adbee3ca-c344-48e3-b000-fc4d47e82937">

**Comment by sf88ux** [comment]
![Screenshot from 2024-09-14 14-50-13](https://github.com/user-attachments/assets/2f8872fd-5f3f-4e16-8857-8380c6941736)
问下，插入图片后，图片本身和标题（表2 全球****）中间的空白特别大，看上去不太好，能解决一下吗？
